### ISSUE TITLE
The secret wasn't in the body

### ISSUE THREAD

**Issue by reiniertc** [the-paperless-project/paperless#645]
I've configured the mail importer in my paperless-docker setup on my synology.
In the log, In 90% of the time I keep getting "The secret wasn't in the body" and the consumer won't import the mail. But the secret is mentioned in the body of the mail.

I've tried:
- only the secret in the body
- the secret as the first word in the body

Any ideas why I keep getting the error?

**Comment by Tooa** [comment]
Can we assume that you have `PAPERLESS_EMAIL_SECRET` configured as an environment variable e.g. either in `paperless.conf` or via `docker-compose.yml`?

Do you have spelled it the same way (e.g. match casing etc.) in your mail as you configured it?

You could add a log statement [here](https://github.com/the-paperless-project/paperless/blob/2cd077d12d1f3046a02b9ed96f3acb4ac30e25cd/src/documents/mail.py#L61) and verify what's actually received as body.

**Comment by reiniertc** [comment]
Yes I have `PAPERLESS_EMAIL_SECRET` configured as an environment variable (in the Synology GUI). And it's spelled the correct way (inlcuding matching cases). 

I'm relatively new to docker: I'm guessing you suggest to add some code to start logging the mail subject, but I have no idea how to do that.

**Comment by drbayer** [comment]
I ran into the same error message. The problem is in the message encoding. When sent using Office 365 the message body is base64 encoded and paperless doesn't read the encoding header for the body. When sent using a service like gmail that does not base64 encode the body, it works properly.
